dcos/util.py

```diff
@@ -92,6 +92,22 @@ def ensure_dir(directory):
                 'Cannot create directory [{}]: {}'.format(directory, e))
 
 
+def ensure_file(path):
```

Comment:
This isn't descriptive enough alone, perhaps `ensure_file_exists` would be better!